app/src/main/res/values/dimens.xml

```diff
@@ -10,6 +11,7 @@
     <dimen name="spacing_large">32dp</dimen>
     <dimen name="spacing_larger">48dp</dimen>
     <dimen name="spacing_extra_large">64dp</dimen>
+    <dimen name="spacing_largest">200dp</dimen>
```

Comment:
This is arbitrary

Reply:
should I replace it with spacing_extra_extra_large or any other suggestion

Reply:
How many times is it used? If it is used in a specific case, then replace name with that case

Reply:
Currently it is used at 3 different places to specify the height of linear layout and appBar layout. I think renaming to spacing_extra_extra_large will be better so that it can be used in future also

Reply:
It's too large. Name in spacing_200